web/src/main/webapp/WEB-INF/pages/candidate_create.jsp

```diff
@@ -68,11 +69,10 @@
                                     code="candidate.name.label"/></label>
 
                             <div class="form-group col-lg-4 col-md-4 col-sm-4">
-                                <input type="text" class="input form-control " name="name" id="name" data-bind="valueWithInit: 'name'"
-                                       value="<c:out value="${candidate.name}"/>"
+                               <spring:message code="candidate.error.name.empty" var="messValue"/> <input type="text" class="input form-control " name="name" id="name" data-bind="valueWithInit: 'name'"
```

Comment:
Use better naming. `messValue` does not tell that it contains the "empty name" error message.